lib/utils/Thing.js

```diff
@@ -195,6 +196,22 @@ export class Thing {
 		return this.isComment() && !!this.element.parentElement && this.element.parentElement.classList.contains('nestedlisting');
 	}
 
+	isSimilarToTitle(value: string): boolean {
```

Comment:
I think this function would make more sense as a standalone `export function isSimilar(a: string b: string)` (or `areSimilar`?) in lib/utils/string.js. That file is exported from /utils as a namespace, so you can call `string.isSimilar(thing.getTitle(), mediaOptions.title)`. `Thing` is already a pretty big class so I want to keep it focused on things (heh) that are strictly related to `thing`s.

Reply:
Yeah I wasn't really sure what the best place for it would be, but that seems a lot more fitting.  I'll add another commit moving it to string.js .